lnrpc/rpc.proto

```diff
@@ -2505,6 +2512,12 @@ message ChannelGraphRequest {
     channels, and public channels that are not yet announced to the network.
     */
     bool include_unannounced = 1;
+
+    /**
+    Whether to calculate node centrality values. Currently this will only
+    calculate node betweenness centrality.
+    */
+    bool calculate_node_centrality = 2;
```

Comment:
name suggestion: just `node_centrality`?

Reply:
This option is now removed in favor of the new GetNodeMetrics RPC call which doesn't have this option. PTAL